Source/FakeItEasy.Specs/UnconfiguredFakeSpecs.cs

```diff
@@ -6,23 +6,28 @@
 
     public static class UnconfiguredFakeSpecs
     {
+        public interface IFoo
+        {
+            bool IsADummy { get; }
+        }
+
         [Scenario]
         public static void VirtualMethod(
             MakesVirtualCallInConstructor fake)
         {
-            "when faking a class with a virtual method"
+            "When faking a class with a virtual method"
                 .x(() => fake = A.Fake<MakesVirtualCallInConstructor>());
 
-            "it should return a default value when the method is called during the constructor"
+            "It should return a default value when the method is called during the constructor"

```

Comment:
@blairconrad no, it's not the primary concern, just one effect of it being so long :smile: 

I don't mind the one long step, but I'll bike shed this once more and say that I think it should be a "When".

"Given" sets up preconditions, "When" invokes the system under test and "Then" assesses the outcome. In this case I''d argue the preconditions are provided by type declaration and our invocation is creating the fake.